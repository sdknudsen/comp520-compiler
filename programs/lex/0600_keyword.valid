break
case
continue
default
else
for
func
if
package
return
struct
switch
type
var
