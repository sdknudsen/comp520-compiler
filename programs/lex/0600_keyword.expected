BREAK
SEMICOLON
CASE
CHAN
CONST
CONTINUE
SEMICOLON
DEFAULT
DEFER
ELSE
FALLTHROUGH
SEMICOLON
FOR
FUNC
GO
GOTO
IF
IMPORT
INTERFACE
MAP
PACKAGE
RANGE
RETURN
SEMICOLON
SELECT
STRUCT
SWITCH
TYPE
VAR
EOF
